### ISSUE TITLE
Tab Tray A11y - Tabs Tray announces Add Tab Button first

### ISSUE THREAD

**Issue by darkwing** [mozilla-mobile/fenix#11541]
## Steps to reproduce
1. Turn on talkback
1. Open Fenix, open tab tray
1. Hear "Add tab.  Open Tabs"

### Expected behavior
1. Hear "Open Tabs" only

### Actual behavior
1. Hear "Add tab.  Open Tabs"

**Comment by mcarare** [comment]
@darkwing This happens because all other views are "offering" a11y info before tab layout does. The most straightforward option to fix this is to declare all other views as not important for a11y, with no initial text, and add importance for accessibility after a tab has been selected.

**Comment by sblatz** [comment]
Wondering if @eeejay has a suggestion on how to "order" these better?

**Comment by mcarare** [comment]
IMO it's not a traversal order problem, but the problem is that the tabs don't exist in the tree at first and are added later than the other items.

**Comment by eeejay** [comment]
The part not written in the STR that I assume is that this happens when zero tabs are open. When I open the tabs tray, the cursor is on the "open tabs" tab, and I hear "Open tabs, Selected. Add tab".

The outlier is "Add tab", since it is a button that is not under the TalkBack cursor. I noticed that the floating circle button is replaced by a toolbar button for "add tab" when TalkBack is on. Is that intentional? Can it have something to do with that behavior?

In other words:
 * I expect, from where the TalkBack cursor is, that the user will hear "Open Tabs, selected".
 * In a zero tabs panel, I would aspire to have the "add tab" button get initial TalkBack cursor (aka. initial focus).
